configs/phalcon.json

```diff
@@ -1,17 +1,8 @@
 {
   "index_name": "phalcon",
   "start_urls": [
-    {
-      "url": "https://phalc.one/docs/(?P<version>.*?)",
-      "variables": {
-        "static_variable": [
-          "3.1",
-          "3.2",
-          "3.3",
-          "3.4"
-        ]
-      }
-    }
+    "https://phalc.one/docs/",
+    "https://phalc.one/docs/3.4/introduction"
```

Comment:
So we don't index other versions? And as `/docs/` redirects to the most recent version, why don't we simply index this one?

Reply:
We do but we extract the version from the meta

Reply:
I understand we extract the version from the `docsearch:version` meta, but why do we need to set `"https://phalc.one/docs/3.4/introduction"` as part of the start urls?

`"https://phalc.one/docs/"` redirects to the latest version (today it's 3.4, but tomorrow it could be 3.5). I would think it's enough to put only `"https://phalc.one/docs/"` in the start urls. Why do we need to put the other one?

Reply:
Thanks to their sitemap we do not need it anymore.
It was required because https://phalc.one/docs/ was not available

Reply:
Ah yes, I understand now. `./docs` is a 404, so we cannot crawl anything from there. But we need it as part of the `start_urls` because it's used as a reference to see if we should index a page or not. And we need to pass the `./3.4/introduction` to have an actual entry point to crawl.

Thanks!